client/connector/HarnessConnector.cs

```diff
@@ -56,7 +56,7 @@ private static HttpClient CreateHttpClientWithTls(Config config, ILoggerFactory
                 return new HttpClient();
             }
 
-#if (NETSTANDARD || NET461)
+#if (NETSTANDARD || NET461 || NET48)
```

Comment:
This is just ensuring that users of .NET 4.8 are warned that on prem requires .NET 5.0